README.md

```diff
@@ -106,6 +107,17 @@ For a comprehensive list with many more clients along with screenshots, the
 - [Matrix CRDT](https://github.com/yousefED/matrix-crdt) - Integrates CRDTs like
   [Yjs](https://github.com/yjs/yjs) with Matrix for storage and sync. `MPL-2.0`
   `TypeScript`
+  
+## Collaborative Maps
+
+- [ODIN](https://odin.syncpoint.io) - [Open Source](https://github.com/syncpoint/ODINv2)
+  Defence Command and Control Information System (C2IS):
+  It's built on [Open Layers](https://openlayers.org), React and [Electron](https://electronjs.org)
+  and uses Matrix in order to replicate the state of the symbols (MIL-STD-2525C) drawn on the map.
+  ODIN is an offline-first application and uses LevelDB to persist state locally. At the lowest 
+  level LevelDB operations are replicated. `AGPL-3.0` `Javascript`
+  Since Matrix' general purpose Javascript SDK does not match the needs of ODIN (i.e. focus 
+  synchronisation on a selected space and its rooms) there is a bespoke [Matrix Client API](https://github.com/syncpoint/matrix-client-api). `MIT` `Javascript`
```

Comment:
This is effectively its own additional entry, so I'd suggest adding this to the SDKs category.